code/game/machinery/Beacon.dm

```diff
@@ -15,7 +15,8 @@
 	Beacon = new /obj/item/radio/beacon(T)
 	Beacon.invisibility = INVISIBILITY_MAXIMUM
 
-	hide(!T.is_plating())
+		if(T)	// TODO Change to happen during initialize
+			hide(!T.is_plating())
```

Comment:
This is in Initialize now, do we still need it?

Reply:
No, it should be OK without.